### DISCUSSION TITLE
Deployment Error on Vercel

### DISCUSSION THREAD

**Discussion by MuhammedFaraz** [ykdojo/defaang#302] [Q&A] [answered]
I've installed Vercel on my forked repo . I have trouble deploying . Deployment fails every time , I have no idea why this is happening. I've also tried to setup the whole damn thing again . Can somebody please help me out??
![Screenshot (54)](https://user-images.githubusercontent.com/92734739/195847138-06083cca-46c8-4ad1-add6-a5c728eb4cf5.png)

**Comment by subhoghoshX** [comment, marked as answer]
Looks like you applied a custom root directory and it can't find the package.json file probably which lists Next.js as a dependency.

**Reply by MuhammedFaraz** [reply]
I had set it to public. I'm trying again what should I choose. And also should I override in Build and Output settings?

![image](https://user-images.githubusercontent.com/92734739/195908402-803544aa-f741-475b-8b18-956300e585da.png)

**Reply by subhoghoshX** [reply]
Choose neither public nor src, just continue while the root directory is selected. And no need to change the build and output settings.
